Classes/Controller/TeaController.php

```diff
@@ -18,18 +19,21 @@ class TeaController extends ActionController
      */
     private $teaRepository;
 
-    public function injectTeaRepository(TeaRepository $teaRepository): void
+    public function injectTeaRepository(TeaRepository $teaRepository): ResponseInterface
```

Comment:
The inject method does not return anything, it is for injecting a dependency. Is there a reason not using constructor injection?

Reply:
the inject method never has to return a ResponseInterface I suppose.
can be the old value void
[3:05](https://typo3.slack.com/archives/CSQ8T0VS5/p1664395534686369)
only actions return a ResponseInterface afaik

Reply:
inject methods should be return type void and not returning anything. Does not make any sense anyway.

Reply:
yeah, I was kind of blind